tests/integration/s2n_client_endpoint_handshake_test.py

```diff
@@ -81,17 +82,32 @@ def try_client_handshake(endpoint, arguments, expected_cipher):
 
     return 0
 
-def well_known_endpoints_test(use_corked_io):
+def well_known_endpoints_test(use_corked_io, tls13_enabled):
+
     arguments = []
-    msg = "\n\tTesting s2n Client with Well Known Endpoints:"
+    msg = "\n\tTesting s2n Client with Well Known Endpoints"
+    opt_list = []
+
+    if tls13_enabled:
+        arguments += ["--tls13", "--ciphers", "default_tls13"]
+        opt_list += ["TLS 1.3"]
     if use_corked_io:
-        msg = "\n\tTesting s2n Client with Well Known Endpoints using Corked IO:"
         arguments += ["-C"]
-    print(msg)
+        opt_list += ["Corked IO"]
+
+    if len(opt_list) != 0:
+        msg += " using "
+        if len(opt_list) > 1:
+            msg += ", ".join(opt_list[:-2] + [opt_list[-2] + " and " + opt_list[-1]])
+        else:
+            msg += opt_list[0]
+
+    print(msg + ":")
 
-    maxRetries = 5
+    maxRetries = 5;
```

Comment:
";" ?

Reply:
Must have been my C instincts kicking in! Luckily Python doesn't care either way, but I'll remove it anyways for clarity.